.gitignore

```diff
@@ -0,0 +1,11 @@
+*.md~
```

Comment:
I have the feeling that you could simply do *.*~ here instead of defining all file types that have this temporary file at the end.